tmt/cli.py

```diff
@@ -502,6 +502,9 @@ def import_(
 @click.option(
     '--format', 'format_', default='yaml', show_default=True, metavar='FORMAT',
     help='Output format.')
+@click.option(
+    '--identifiers', is_flag=True,
```

Comment:
Thinking now a bit about the option name: As for now we keep everything in singular form, so shall it be rather `--identifier`? Or, perhaps, to be a little bit more specific, should we call it `--fmf-id`?

Reply:
Either way seems reasonable. @lukaszachy do you have an opinion on this one?

Reply:
Hm, good questions - I had wrote long reply first, deleted it and now I'm wring the complete opposite ;)

Let's go with `--fmf-id` because it is reported under this name in `tmt tests show -v` output.

Reply:
@juk, are you going to update the PR with the proposed changes? It would be nice to get it into `1.7` which we (hopefully) release tomorrow.

Reply:
@psss will return from vacation this weekend and update the PR the week of Aug 16-20

Reply:
Great, thanks! The `1.7` slipped again, the next tentative release date is Tue, Aug 17 :)